custom/ewsghana/reports/specific_reports/stock_status_report.py

```diff
@@ -73,21 +73,27 @@ def charts(self):
             def convert_product_data_to_stack_chart(rows, chart_config):
                 ret_data = []
                 for k in ['Stocked out', 'Not Stocked out', 'No Stock Data']:
+
+                    def calculate_percent(x, y):
+                        return float(x) / float((y or 1))
```

Comment:
@millerdev 

```y``` couldn't be string.

I added ```(y or 1)``` just to be sure they do not get ZeroDivisionError.